docs/rest-api/rest-zeppelin-server.md

```diff
@@ -33,6 +33,46 @@ If you work with Apache Zeppelin and find a need for an additional REST API, ple
 
 ## Zeppelin Server REST API list
 
+### Get Zeppelin version
+  <table class="table-configuration">
+    <col width="200">
+    <tr>
+      <td>Description</td>
+      <td>This ```GET``` method returns Zeppelin version</td>
+    </tr>
+    <tr>
+      <td>URL</td>
+      <td>```http://[zeppelin-server]:[zeppelin-port]/api/version```</td>
+    </tr>
+    <tr>
+      <td>Success code</td>
+      <td>200</td>
+    </tr>
+    <tr>
+      <td> Fail code</td>
+      <td> 500 </td>
+    </tr>
+    <tr>
+      <td> sample JSON response
+      </td>
+      <td>
+        <pre>
+{
+  "status": "OK",
+  "message": "Zeppelin version",
+  "body": [
+    {
+      "version": "0.8.0-SNAPSHOT",
```

Comment:
perhaps don't put `-SNAPSHOT` as an example? in case we forget to change this when we make release